react-app/src/components/auth/LoginForm.js

```diff
@@ -44,39 +44,44 @@ const LoginForm = () => {
   return (
     <form onSubmit={onLogin}>
       <div>
-      <Logo />
+        <Logo />
         <h1>Welcome Back!</h1>
-        <h4>Please sign in to continue </h4>
-        <div>
-          {errors.map((error, ind) => (
-            <div key={ind}>{error}</div>
-          ))}
+        <div className='auth-fields'>
+          <div className='auth-field'>
+            <div className='auth-label-err'>
+              <label htmlFor='email'>Email</label>
+            </div>
+            <input
+              className='auth-input'
+              name='email'
+              type='text'
+              placeholder='Please enter your email'
+              value={email}
+              onChange={updateEmail}
+            />
+          </div>
+          <div className='auth-field'>
+            <div className='auth-label-err'>
+              <label htmlFor='password'>Password</label>
+            </div>
+            <input
+              className='auth-input'
+              name='password'
+              type='password'
+              placeholder='Please enter your password'
+              value={password}
+              onChange={updatePassword}
+            />
+          </div>
         </div>
-        <div>
-          <label htmlFor='email'>Email</label>
-          <input
-            className='fInput'
-            name='email'
-            type='text'
-            placeholder='Email'
-            value={email}
-            onChange={updateEmail}
-          />
+        <div id="login-err">
+          {errors.length !== 0 &&
```

Comment:
If we're hiding the specific error message (no such user, or right user wrong password), we should really hide it in the API too since they can just go look at the actual error message in the network tab. I'm gonna approve this either way cause as you said, it really doesn't matter on this site. But if we do a security measure, I'd like to see it done all the way.